### PR TITLE
add test to database

### PR DESCRIPTION
**Pull request by AnatoliAgeichik**
In this PR the operation of the database was tested

### CODE REVIEW

File: app/connect_db.py [hunk 1]
```diff
@@ -21,6 +21,9 @@ def update_language_target_for_user(user_id, language):
 
 
 def add_new_user(user_id, language_target="en", language_from="ru"):
+    print(user_id)
```
Comment (artsiom-streetshares) [inline app/connect_db.py]: this is fine for now. though generally you should be using module for printing purpose, we will be doing it in the next project

File: test/bot_test.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+def test_add_new_user_without_lang():
+    assert test_user_id in get_all_users()
+
+
+def test_add_new_user_with_lang():
+    test_user_id = 123
+    delete_user(test_user_id)
+    lang_target = "fr"
+    lang_from = "en"
+    add_new_user(test_user_id, lang_target, lang_from)
```
Comment (artsiom-streetshares) [inline test/bot_test.py:24]: a bit more concise without readability loss
```suggestion
    add_new_user(test_user_id, lang_target="fr", lang_from="en")
```

File: test/bot_test.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    delete_user(test_user_id)
+
+
+def test_add_new_user_without_lang():
+    assert test_user_id in get_all_users()
+
+
+def test_add_new_user_with_lang():
+    test_user_id = 123
+    delete_user(test_user_id)
```
Comment (artsiom-streetshares) [inline test/bot_test.py:21]: a good practice is to setups your tests in such a way that they leave database clean, so you don't need to assume that this test or any other has left anything that you need to cleanup. we will be doing that in subsequent projects.

**Review by artsiom-streetshares — APPROVED** [review]

File: test/bot_test.py (hunk 2 above)
Comment (AnatoliAgeichik) [reply test/bot_test.py:24]: I still use these variables in assert

Comment (artsiom-streetshares) [reply test/bot_test.py:24]: ok then. though in tests it would be fine to use constant strings in assert as well.

**Pull request merged**